templates/zerver/help/index.md

```diff
@@ -1,18 +1,17 @@
 # Zulip User Documentation
 
 Zulip is a group chat app. Its most distinctive characteristic is that
-conversation within an organization can be divided into “**streams**”
+conversation within an organization is divided into “**streams**”
 and further subdivided into “**topics**”, so that
 much finer-grained conversations are possible than with IRC or other
 chat tools.
 
 Most people use Zulip on the Web. There are also mobile apps for
-Android/iOS, and desktop apps for Mac/Linux/Windows, as well as a
-cross-platform version and a version for Platform 9. See
-[the documentation on apps](/apps) for more information.
+Android/iOS, and desktop apps for Mac/Linux/Windows, which can be
+downloaded [here](/apps).
```

Comment:
@gnprice I think these `/apps` links should be moved to absolute, pointing at zulipchat.com, right?